misc/wsdl-to-ballerina/src/main/java/io/ballerina/wsdltoballerina/cmd/ErrorMessages.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.wsdltoballerina.cmd;
+
+public class ErrorMessages {
+    public static final String MISSING_WSDL_PATH = "A WSDL definition path is required to generate the " +
+            "client. \ne.g: bal openapi --input <WSDL>";
```

Comment:
Use the correct command name.
```suggestion
    public static final String MISSING_WSDL_PATH = "A WSDL definition path is required to generate the " +
            "client. \ne.g: bal wsdl --input <WSDL>";
```